### PR TITLE
[AMP] Add BatchNorm_1D_2D_3D skip for paddle.amp.decorate

### PR DESCRIPTION
**Pull request by zhangbo9674**
### PR types
Bug fixes

### PR changes
APIs

### Describe
框架的`batch_norm` op参数不可以为fp16，`paddle.amp.decorate`新增对`paddle.nn.BatchNorm_1D_2D_3D`的判断，上述三个网络将跳过decorate实现的参数改写。

### CODE REVIEW

**Review by zhiqiu — DISMISSED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
LGTM
